support/core.go

```diff
@@ -184,3 +185,10 @@ func GetNodeInternalIP(t Test, node corev1.Node) (IP string) {
 
 	return
 }
+
+func ResourceName(t Test, obj any) string {
```

Comment:
AFAIK the function can have just one argument - the struct used to retrieve its field from.
Otherwise it can't be used in `WithTransform` function

Reply:
You're right! Thanks Karel - Fixed.